### PR TITLE
Nightwatch

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review

* end-to-end tests with nightwatch

#15

### CODE REVIEW

File: examples/direct-upload/test/e2e/upload.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  'Image Upload': function (browser) {
+    browser
+      .url('http://localhost:8000')
+      .waitForElementVisible('body', 1000)
+      .waitForElementVisible('input#fileInput', 1000)
+      .pause(1000)
+      .setValue('input#fileInput', require('path').resolve(__dirname + '/testfile.png'))
```
Comment (nelsonic) [inline examples/direct-upload/test/e2e/upload.js]: Any _reason_ to not be using a [`cute kitten`](https://www.google.pt/search?q=cute+kitten&tbm=isch) here...? 😜

**Comment by nelsonic** [comment]
@jackcarlisle looks good. would you mind adding a couple of lines to the https://github.com/dwyl/image-uploads/blob/master/examples/direct-upload/README.md informing people that we are using Nightwatch for front-end tests ?

**Comment by jackcarlisle** [comment]
@nelsonic readme updated, cute kitten added 😉

**Comment by nelsonic** [comment]
@jackcarlisle amaze! 🎉

**Pull request merged**